### PR TITLE
Add colouring page

### PR DESCRIPTION
**Pull request by ginifizz**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

Adds a category "colouring webby" to website resources.

image

### CODE REVIEW

File: src/pages/resources/colouring-webby.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+export const query = graphql`
+  query {
+    allFile(filter: { sourceInstanceName: { eq: "colouring" } }) {
+      nodes {
+        name
+        childImageSharp {
+          original {
+            src
+            height
+            width
```
Comment (alanpoulain) [inline src/pages/resources/colouring-webby.js]: Needed?

File: src/pages/resources/colouring-webby.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          }
+          mini: resize(width: 500, quality: 100) {
+            src
+            width
+            height
```
Comment (alanpoulain) [inline src/pages/resources/colouring-webby.js]: Needed?

File: src/pages/resources/colouring-webby.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        }
+      }
+    }
+  }
+`;
+
+const imageType = PropTypes.shape({
+  src: PropTypes.string,
+  width: PropTypes.number,
+  height: PropTypes.number,
```
Comment (alanpoulain) [inline src/pages/resources/colouring-webby.js]: Needed?

File: src/pages/resources/colouring-webby.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            height
+          }
+        }
+      }
+    }
+  }
+`;
+
+const imageType = PropTypes.shape({
+  src: PropTypes.string,
```
Comment (alanpoulain) [inline src/pages/resources/colouring-webby.js]:
```suggestion
  src: PropTypes.string.isRequired,
```

**Comment by jewome62** [comment]
https://api-platform.com/resources/colouring-webby/

**Pull request merged**